lib/fog/vsphere/requests/compute/create_vm.rb

```diff
@@ -5,6 +5,7 @@ class Real
         def create_vm(attributes = {})
           # build up vm configuration
 
+          pp attributes
```

Comment:
I believe this debug code can safely be deleted.

Reply:
Forgot that one :disappointed: thanks :+1: